messaging/src/main/java/org/axonframework/serialization/avro/AvroConverterConfiguration.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.serialization.avro;
+
+import jakarta.annotation.Nonnull;
+import org.apache.avro.message.SchemaStore;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Configuration for the AvroConverter.
+ *
+ * @param strategies                                converter strategies to support different representation of objects
+ *                                                  (SpecificRecordBase, Avro4K).
+ * @param includeDefaultAvroSerializationStrategies flag to include the default strategies.
+ * @param schemaStore                               schema store to resolve Avro schemas.
+ * @param schemaIncompatibilityChecker              schema incompatibility checker.
+ * @param avroConverterStrategyConfiguration        configuration for Avro converter strategy.
+ */
+public record AvroConverterConfiguration(
+        @Nonnull List<AvroConverterStrategy> strategies,
+        boolean includeDefaultAvroSerializationStrategies,
```

Comment:
Nit: I think we should rename this parameter:
```suggestion
        boolean includeDefaultAvroConverterStrategies,
```